### PR TITLE
Expose config and pathBuilder.

### LINKED ISSUE #48: Expose config and pathBuilder.
It would be useful in some situations to be able to access the `config and `pathBuilder` objects within consuming projects in order to reduce duplication of these values.

### PR DESCRIPTION
**Pull request by DamianMullins**
Resolves #48.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,72 +9,105 @@ Gulp build tasks for use across Fozzie modules.
...
+```
 
-## Setup
+If you do not add a `.babelrc` file (you may be writing es5 code for example) then the code will be bundled up as is.
 
-To integrate these tasks into your project you need to complete the following steps:
+#### JavaScript Linting
 
-Add `@justeat/gulp-build-fozzie` to your project
+Add an `.eslintrc` file to the root of your project which the following content to use the JS linting rules we recommend when running the [`scripts:lint`](#scriptslint) task:
```
Comment (ashleynolan) [inline README.md]: Think `which` should be `with`?

File: README.md [hunk 2]
```diff
@@ -9,72 +9,105 @@ Gulp build tasks for use across Fozzie modules.
...
 
-```bash
-yarn add @justeat/gulp-build-fozzie
+```json
+{
+    "extends": "@justeat/eslint-config-fozzie"
```
Comment (ashleynolan) [inline README.md]: Is it worth mentioning that these rules can be extended if needed and used as a base?

File: README.md [hunk 3]
```diff
@@ -9,72 +9,105 @@ Gulp build tasks for use across Fozzie modules.
...
 
-build(gulp, /*options*/);
+```json
+"stylelint": {
+    "extends": "@justeat/stylelint-config-fozzie"
+}
 ```
 
-That's it! You can now run any of [the Gulp tasks](#the-gulp-tasks).
+If you wish to extend these rules, you can also define your own `.stylelintrc` file in the root of your project with the rules that you wish to override.
```
Comment (ashleynolan) [inline README.md]: This is a alternate way of specifying the config.  You can extend the rules in the `package.json` too (like [here on uk ConsumerWeb](https://github.je-labs.com/CWA/ConsumerWeb/blob/master/src/ConsumerWeb/package.json#L50)).

Probably worth [linking to this](https://stylelint.io/user-guide/configuration/) as this defines the order of priority in whcih stylelint looks for config.

**Comment by ashleynolan** [comment]
👏 👏 👏 👏  on the pathbuilder stuff – very nice 🎯

**Comment by DamianMullins** [comment]
@ashleynolan I've updated the docs are per your comments 👍

**Pull request merged**